pkg/nvmdev/nvmdev.go

```diff
@@ -321,21 +321,17 @@ func (m *Device) Delete() error {
 }
 
 // GetPhysicalFunction gets the physical PCI device backing a 'parent' device.
-func (p *ParentDevice) GetPhysicalFunction() (*nvpci.NvidiaPCIDevice, error) {
-	if !p.IsVF {
-		return p.NvidiaPCIDevice, nil
+func (p *ParentDevice) GetPhysicalFunction() *nvpci.NvidiaPCIDevice {
+	if p.IsVF() {
+		return p.SriovInfo.VirtualFunction.PhysicalFunction
 	}
 
```

Comment:
nit: remove the newline here since this is such a simple function

Reply:
removed